### PR TITLE
update to latest iris

### PR DESCRIPTION
**Pull request by hiveminded**

### CODE REVIEW

**Comment by kinosang** [comment]
`godep restore` failed. please try to fix it.

**Comment by hiveminded** [comment]
Yes, missing the kataras/golog I'll fix it, but I will remove the travis for go < 1.9 in order that to work

File: .travis.yml [hunk 1]
```diff
@@ -5,12 +5,11 @@ language: go
 matrix:
   fast_finish: true
   include:
+    - go: tip
```
Comment (kinosang) [inline .travis.yml]: add `go: 1.9` here

Comment (kinosang) [inline .travis.yml:8]: add `go: 1.9` here

File: .travis.yml [hunk 2]
```diff
@@ -5,12 +5,11 @@ language: go
...
+  allow_failures:
```
Comment (kinosang) [inline .travis.yml:9]: remove those versions instead of moving them to `allow_failures `

**Review by kinosang — CHANGES_REQUESTED** [review]
changes required

**Pull request merged**